client/src/components/NewEventType/NewEventType.tsx

```diff
@@ -0,0 +1,109 @@
+import { Formik } from 'formik';
+import useStyles from './useStyles';
+import { Grid, Box, TextField, Typography, Button, Paper } from '@material-ui/core';
+import NavBar from '../NavBar/NavBar';
+import createEvent from './../../helpers/APICalls/createMeeting';
+import { Event } from '../../interface/Event';
+import { useHistory } from 'react-router-dom';
+
+const defaultFormValues = {
+  eventName: '',
+  eventDescription: '',
+  eventDuration: '',
+  eventUrl: '',
+};
+interface InitialFormValuesTypes {
+  eventName: string;
+  eventDescription: string;
+  eventDuration: string;
+  eventUrl: string;
+}
+interface Props {
+  initialFormValues?: InitialFormValuesTypes;
+}
+const NewEventType = ({ initialFormValues = defaultFormValues }: Props): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+
+  const handleFormSubmit = ({ eventName, eventDescription, eventUrl, eventDuration }: Event) => {
+    createEvent({ eventName, eventDescription, eventUrl, eventDuration }).then((data) => {
+      if (data.success) {
+        history.push('/dashboard');
+      }
+      //TODO catch error
```

Comment:
I don't understand why not just catch the error? Is there a reason we need to have a TODO here?

Reply:
I was thinking, maybe we would want to render according to loading error states. I mean we would have something below:
`const { loading, error, data} = useQuery(apiCallFunction)`

After that we would condionally render. But I forgot to mention this in yesterday's meeting

Reply:
fixed